### ISSUE TITLE
Virus detected - unable to download

### ISSUE THREAD

**Issue by Pi-Mania** [YoMizzy/PSXBINq#1]
Looks useful but getting Virus detected - unable to download. (Chrome/IE/Edge Win10

**Comment by YoMizzy** [comment]
Yeah, I've noticed that. there is no virus just unsigned code making windows freak out. working on fixing the issues. using a diff exe compiler.

**Comment by turbocomppro** [comment]
Any update on this?  Still getting virus warning.

**Comment by YoMizzy** [comment]
I uploaded the raw code via Windows Batch file, should be able to run without any issues now.

**Comment by YoMizzy** [comment]
https://github.com/YoMizzy/PSXBINq/releases/tag/1.0
